src/components/QuickFacts.js

```diff
@@ -47,13 +46,13 @@ export const QuickFacts = ({ article, goToArticleSubpage, dir, close, closeAll }
     article.contentLang,
     containerRef,
     linkHandlers,
-    [scrollPosition, textSize],
+    [scrollPosition],
```

Comment:
Not sure why this has been removed, changing the font size does change which links are currently visible so it needs to recompute.

Reply:
This was needed before because the user was able to change the font size from within `Quickfacts` by directly pressing 4, 5, 6 but now that's not the case anymore